pyproject.toml

```diff
@@ -81,9 +81,18 @@ validators = [
     "pandas"
 ]
 
+[tool.isort]
+profile = "black"
+
 [tool.pylint.'MESSAGES CONTROL']
 extension-pkg-whitelist = "pydantic"
 
+[tool.pytest.ini_options]
+testpaths = [
+    "tests",
+    "parsers/test"
+]
+
```

Comment:
Prevents duplicate runs of the tests by limiting the scope to the correct folders.
Before all tests would run from both the `parsers/test` folder and `electricitymap/contrib/parsers/test` as the later was a symlink to the former.